src/components/Display.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Display = () => (
+  <h1>
+    0
+  </h1>
+);
```

Comment:
- [ ] The display component should accept `the result of the calculation` as an incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png))

- [ ] Also, its result prop should be a String with default value = 0 ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)).

- Kindly fix these

Reply:
Fixed